patches/third_party/devtools-frontend/src/front_end-panels-application-ApplicationPanelSidebar.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/front_end/panels/application/ApplicationPanelSidebar.ts b/front_end/panels/application/ApplicationPanelSidebar.ts
+index 5d024c6d3ce493ae4df798593d4a22f84e4acf2d..ce84d171b90d9f7f9363a90513236ab34fca2b06 100644
+--- a/front_end/panels/application/ApplicationPanelSidebar.ts
++++ b/front_end/panels/application/ApplicationPanelSidebar.ts
+@@ -355,7 +355,7 @@ export class ApplicationPanelSidebar extends UI.Widget.VBox implements SDK.Targe
+     storageTreeElement.appendChild(this.cookieListTreeElement);
+ 
+     this.trustTokensTreeElement = new TrustTokensTreeElement(panel);
+-    storageTreeElement.appendChild(this.trustTokensTreeElement);
++    //storageTreeElement.appendChild(this.trustTokensTreeElement);
```

Comment:
@petemill Do you think calling storageTreeElement.removeChild(this.trustTokensTreeElement); would be better than this?

Reply:
I can't think why adding and removing is better than just not adding it in the first place. Let me know if you have thought of a reason.

Reply:
Oh, NVM then. I thought I've seen something like that in settings page. That's why I asked 😄

Reply:
@petemill can we avoid patching here by overriding `TrustTokensTreeElement` and just making it an empty div or something?

Reply:
There is no mechanism to override TrustTokensTreeElement or any piece of TS/JS here without a patch. A more complex one I imagine.

Reply:
@sangwoo108 please change this comment to be a single line added above `this.trustTokensTreeElement = new TrustTokensTreeElement(panel);` with `/*` and a single line added below `storageTreeElement.appendChild(this.trustTokensTreeElement);`
We don't want to create `TrustTokensTreeElement` if we're not going to use it and adding lines is better than modifying them for patches

Reply:
Good call! done.

Reply:
> [2023-03-29T00:54:07.410Z] ../../third_party/devtools-frontend/src/front_end/panels/application/ApplicationPanelSidebar.ts(253,3): error TS2564: Property 'trustTokensTreeElement' has no initializer and is not definitely assigned in the constructor.


Because of this error, added one more line to the patch file.